Add Resource Drivers section  to extending Kubernetes documentation (#43742)
kubernetes/website: content/en/docs/reference/glossary/PodSchedulingContext.md

---
id: PodSchedulingContext
title: PodSchedulingContext
full-link: /docs/concepts/scheduling-eviction/dynamic-resource-allocation/
date: 2023-11-28
short_description: >

>  A short-lived object that is created by the kube-scheduler to coordinate with resource drivers the
>  selection of a Node for the Pod, that uses one or more ResourceClaims.

Reviewer: ```suggestion
  A short-lived object used to facilitate Pod scheduling and resource allocation.
```

related:
 - kube-scheduler
 - resource-claim
 - resource-driver
---

>  A [Pod Scheduling Context](/docs/concepts/scheduling-eviction/dynamic-resource-allocation/) is used

Reviewer: ```suggestion
A [PodSchedulingContext](/docs/concepts/scheduling-eviction/dynamic-resource-allocation/) is used
```
Is there a fragment we could link to?

>  by kube-scheduler when Pod needs ResourceClaims in order to be scheduled.

Reviewer: Please use a glossary tooltip for “scheduled”; it'll help readers.

<!--more-->

> Resource drivers and kube-scheduler communicate through RecourceClaim and PodSchedulingContext objects

Reviewer: ```suggestion
Resource drivers and kube-scheduler communicate through ResourceClaim and PodSchedulingContext objects
```
